### PR TITLE
Updates for Azure VM support

### PR DESCRIPTION
**Pull request by Cyber-JiuJiteria**
Updates:

- /src/core/settings.py -> include the static_root for the UI static files
- Dockerfile to include the gunicorn command for executing
- readme.md to include information for the public facing development environment

### CODE REVIEW

**Comment by glenda1015** [comment]
Looks goo to me!

File: README.md [hunk 1]
```diff
@@ -55,6 +55,26 @@ Issues enabling python virtualenv
...
+  * Install the following and any of their dependencies
+    * [nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-22-04)
+    * [docker](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-22-04)
+    * [docker compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-22-04)
+    * [gunicorn](https://docs.gunicorn.org/en/stable/install.html)
+  * Update `/etc/nginx/nginx.conf` and `/etc/nginx/sites-available/default` file with the respected settings
+  * Collecting the static files for UI
+    * Ensure that the `/opt/omega/static` directory is available, if not, create it.
+    * Enter the triage-portal container ( `docker exec -it omega-triage-portal /bin/bash` ) and run `python manage.py collectstatic`
+    * Move the static files to the `/opt/omega/static` directory
```
Comment (JLLeitschuh) [inline README.md:76]: Why isn't this handled automatically? Also, what if the static files change, but the VM isn't rebuilt?

File: README.md [hunk 2]
```diff
@@ -55,6 +55,26 @@ Issues enabling python virtualenv
...
+In the event of a virtual machine having to be destroy and a new one taken it's place, here are important details when deploying a new proof of concept using Azure VM
+
+* In the Networking settings for the Azure VM, ensure that HTTP, HTTPS, and SSH are configured in the firewall.
+* In the VM,
+  * Perform a git pull via HTTP git clone URI in the `/app` directory
+  * Install the following and any of their dependencies
+    * [nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-22-04)
+    * [docker](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-22-04)
+    * [docker compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-22-04)
+    * [gunicorn](https://docs.gunicorn.org/en/stable/install.html)
```
Comment (JLLeitschuh) [inline README.md:71]: This one looks like it's only needed in the docker container, right? Or it's also needed in the VM?

**Review by JLLeitschuh — APPROVED** [review]
This doesn't look like it adds anything to keep the codebase in sync with the VM the demo is running on. How is that handled? Is there some code on the VM that continuously pulls the latest version from GitHub?

Also, why aren't the `nginx.conf` files checked into source control?

File: README.md (hunk 2 above)
Comment (Cyber-JiuJiteria) [reply README.md:71]: The VM needs these packages installed.

**Comment by JLLeitschuh** [comment]
In the short term, should we check the `nginx.conf` and cron script into source control as well under a `.vm-util` directory in the short term? That way, if the VM gets nuked, then we have them somewhere?

**Pull request merged**